lib/chef/knife/vsphere_vm_state.rb

```diff
@@ -110,6 +110,9 @@ def run
       when 'reset'
         vm.ResetVM_Task.wait_for_completion
         puts "Reset virtual machine #{vmname}"
+      when 'reboot'
+        vm.RebootGuest
```

Comment:
The other options in this command all wait for the command to complete in some manner. Is it advantageous not to do it here?